src/redist/targets/BundledTemplates.targets

```
@@ -20,56 +20,65 @@
 
   </Target>
 
-  <ItemGroup>
+  <ItemGroup Label=".NET 9.0 templates">
+    <Bundled90Templates Include="Microsoft.DotNet.Common.ItemTemplates" PackageVersion="$(MicrosoftDotNetCommonItemTemplates90PackageVersion)" />
+    <Bundled80Templates Include="Microsoft.DotNet.Common.ProjectTemplates.8.0" PackageVersion="$(MicrosoftDotNetCommonItemTemplates80PackageVersion)" />
+    <Bundled80Templates Include="Microsoft.DotNet.Test.ProjectTemplates.8.0" PackageVersion="$(MicrosoftDotNetTestProjectTemplates80PackageVersion)" />
```

Do we want `Bundled80Templates` items in the 9.0 template group?  I imagine we don't have 9.0 templates for these yet, but these items are exactly the same as in the 8.0 template `ItemGroup`, so I don't think we want them repeated.  Probably we should either call them `Bundled90Templates` or comment them out until we have templates of the right version.

```suggestion
    <Bundled90Templates Include="Microsoft.DotNet.Common.ProjectTemplates.8.0" PackageVersion="$(MicrosoftDotNetCommonItemTemplates80PackageVersion)" />
    <Bundled90Templates Include="Microsoft.DotNet.Test.ProjectTemplates.8.0" PackageVersion="$(MicrosoftDotNetTestProjectTemplates80PackageVersion)" />
```